src/wire/ndiscoption.rs

```diff
@@ -402,6 +402,16 @@ pub struct PrefixInformation {
     pub prefix: Ipv6Address,
 }
 
+impl PrefixInformation {
+    /// Validates the prefix information option against check a, b, c in
+    /// https://www.rfc-editor.org/rfc/rfc4862#section-5.5.3
+    pub fn valid_prefix_info(&self) -> bool {
```

Comment:
This probably reads better?
```suggestion
    pub fn is_valid_prefix_info(&self) -> bool {
```